README.md

```diff
@@ -211,7 +211,7 @@ may have the following properties:
   predicted and hardwired into the test case file.
 * `input`: An array with the lines of input (each line being a string)
   that should be fed to the Logstash process.
-* `testcases`: An array of test case hashes, consisting of a field `input`
+* `testcaseset`: An array of test case hashes, consisting of a field `input`
```

Comment:
The struct member is named TestCases and the corresponding JSON field is testcases. This is correct, since it's the whole file that's a testcase set, an entity that can contain one or more references to testcases. RIght?

https://github.com/magnusbaeck/logstash-filter-verifier/blob/c528371c0a085cec8beaa48301b90497cac61c5a/testcase/testcase.go#L66